src/Goteo/Model/Filter.php

```diff
@@ -1380,6 +1380,11 @@ public function getPromoters($offset = 0, $limit = 0, $count = false, $lang = nu
                 AND project.status = :project_status
                 ";
             $values[':project_status'] = $this->project_status;
+        } else if ($this->project_status == Project::STATUS_NEGOTIATION) {
+            $sqlFilter .= "
+                AND project.status = :project_status AND project.`id` NOT REGEXP '[0-9a-f]{32}'
+            ";
+            $values[':project_status'] = Project::STATUS_EDITING;
```

Comment:
These lines are duplicate code of below lines (or the other way around) 1545-1549.

There would be a reduced code maintenance if they would be in a common method. At least the `$sqlFilter` one.